src/components/Homepage-MainActivities.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import ProjectsImage from "./Homepage-Projects";
+import ProjectsText from "./Homepage-ProjectsText";
+import EventsImage from "./Homepage-Events";
+import EventsText from "./Homepage-EventsText";
+import PodcastImage from "./Homepage-Podcasts";
+import PodcastText from "./Homepage-PodcastsText";
+
+import styles from "../css/Homepage-MainActivities.module.css";
+
+export default props => {
+  return (
+    <>
+      <div className={styles.homepageActivity}>
+        <ProjectsImage />
+        <ProjectsText />
+      </div>
```

Comment:
You should probably create an `Activity` component that wraps the div and the Image/Text